e2e/src/editCra.e2e-spec.ts

```diff
@@ -0,0 +1,152 @@
+import { EditCraPage } from './editCra.po';
+
+import { browser, by, ElementFinder } from 'protractor';
+
+describe('When I input my timesheet', () => {
+  let editCraPage: EditCraPage,
+      nameInput: ElementFinder,
+      emailInput: ElementFinder,
+      titleInput: ElementFinder,
+      clientInput: ElementFinder;
+
+  beforeEach(() => {
+    editCraPage = new EditCraPage();
+    editCraPage.navigateTo();
+    browser.waitForAngular();
+
+    nameInput = editCraPage.getInputByLabelText('Nom');
+    emailInput = editCraPage.getInputByLabelText('Email');
+    titleInput = editCraPage.getInputByLabelText('Intitulé');
+    clientInput = editCraPage.getInputByLabelText('Client final');
+  });
+
+  it('should be possible to enter my name', () => {
+    const name = 'Tester';
```

Comment:
Why do you declare a constant that you only use on the line bellow? Do you even know what a constant is?